examples/medplum-chart-demo/src/components/graphs/ObservationGraph.tsx

```diff
@@ -0,0 +1,80 @@
+import { Paper } from '@mantine/core';
+import { formatDate, getReferenceString } from '@medplum/core';
+import { Coding, Observation, ObservationComponent, Patient } from '@medplum/fhirtypes';
+import { useMedplum } from '@medplum/react';
+import { useEffect, useState } from 'react';
+import { LineChart } from './LineChart';
+import { measurementStyles } from './measurement-constants';
+import { ChartData, ChartDataset } from 'chart.js';
+
+interface ObservationGraphProps {
+  code: Coding;
+  patient: Patient;
+}
+
+export function ObservationGraph(props: ObservationGraphProps): JSX.Element {
+  const medplum = useMedplum();
+
+  // Get the chartDataset codes and units
+  const { chartDatasets } = measurementStyles[props.code.display as string];
```

Comment:
instead of making this a direct index to the map, convert this to a getter function: 
`getMeasurementStyles(props.code)`

indexing on the display string is brittle, and this logic might change in the future. Hiding it inside a function allows it to change later